tests/integration/feature_store_integration/test_feature_materialize.py

```diff
@@ -77,8 +77,34 @@ def features_fixture(event_table):
         feature_names=["EXTERNAL_FS_COUNT_OVERALL_7d"],
     )["EXTERNAL_FS_COUNT_OVERALL_7d"]
 
+    # Dict feature
+    cross_aggregate_feature1 = event_view.groupby(
+        "ÜSER ID", category="PRODUCT_ACTION"
+    ).aggregate_over(
+        "ÀMOUNT",
+        method="sum",
+        windows=["7d"],
+        feature_names=["amount_sum_across_action_7d"],
+    )[
+        "amount_sum_across_action_7d"
+    ]
+
+    cross_aggregate_feature2 = event_view.groupby(
+        ["CUST_ID"], category="PRODUCT_ACTION"
```

Comment:
Is there a reason to use a different groupby key here? If using the same key, with different windows the similarity should be non-zero as a sanity check (assuming both windows covered some events).

Reply:
The main reason to use a different groupby key here is to make sure it triggers on demand feature view construction. I could revise this test to use the same groupby key but different feature job setting.

Reply:
I forget that we block the composite feature with different feature job setting. 😓 
This test actually aims to check the integration part for the count dict related operation. The more detailed tests are covered in the unit tests (`tests/unit/query_graph/odfv/test_count_dict.py`). Do you have any suggestion that we can construct the cosine similarity feature that will trigger ODFV construction?

Reply:
I see. I think it's fine to leave this as is if there are already more detailed unit tests.